.gitignore

```diff
@@ -11,7 +11,7 @@
 *.iws
 out/
 .home*/
-work/
+work*/
```

Comment:
since we're fixing this, can we make it `/work*/` so that we can have a _package_ called `work` and not have it be ignored?

For what it is worth, this is no worse than it is right now (in fact it is better)